README.md

```diff
@@ -105,8 +107,9 @@ The following options are supported and closely mimic the options of Async profi
 - `--async-profiler-event`: The event to sample, e.g. `cpu` or `alloc`. Defaults to `cpu`
 - `--async-profiler-count`: The count to use when aggregating event data. Either `samples` or `total`. `total` is especially useful for allocation profiling. Defaults to `samples`
 - `--async-profiler-interval`: The sampling interval in ns, defaults to 10_000_000 (10ms)
+- `--async-profiler-alloc-interval`: The sampling interval in bytes for allocation profiling, defaults to 10 bytes. Corresponds to the `--alloc` command line option for Async profiler.
+- `--async-profiler-lock-threshold`: lock profiling threshold in nanoseconds, defaults to 1ns.
```

Comment:
Does this correspond to some command-line argument for Async profiler?

Also, `1 ns` is more readable for me than `1ns`.

Reply:
It corresponds to `--lock`. I used `1ns` instead of `1 ns`, since we use `10ms` above. I can change that so both use the space.

Reply:
I improved the readme. Please take a look.